docs/usage/from-a-terminal.rst

```diff
@@ -1057,12 +1062,12 @@ Global overview
                                 Configured value: False
         -t LOOKUP__TIMEOUT, --timeout LOOKUP__TIMEOUT
                                 Sets the default timeout to apply to each lookup
-                                utilities everytime it is possible to define a timeout.
+                                utilities every time it is possible to define a timeout.
```

Comment:
Please also apply the change at: https://github.com/funilrys/PyFunceble/blob/0d5e5450473aff6712277d8dd0bbc240e2f953f1/PyFunceble/cli/entry_points/pyfunceble/cli.py#L425

Reply:
Leave hanging, till I have sorted the `everytime` vs `every time`